docs/CONFIG-VARS.md

```diff
@@ -41,8 +41,8 @@ Find details on how to retrieve that information under [Azure Help Topics](./use
 
 | Name | Description | Type | Default |
 | :--- | ---: | ---: | ---: |
-| tenant_id | your Azure tenant id | string  |
-| subscription_id | your Azure subscription id | string  |
+| tenant_id | your Azure tenant id | string  | |
```

Comment:
So for both tenant_id and subscription_id no default? Just asking as client_id and client_secret seem to have values of ""

Reply:
Yes, using defaults from https://github.com/sassoftware/viya4-iac-azure/blob/main/variables.tf#L1

Reply:
got it. thx.